blog/templates/blog/blog.html

```diff
@@ -6,20 +6,23 @@
 {% block blog_content %}
 {% for object in object_list %}
 
-    
     <div class="blog-post">
-	    <h2><a class="blog-post-title"><a href="{% url "entry_view" object.id %}">{{object.title}}</a></h2>
-	    <p class="blog-post-meta"> {{object.posted}} 
-	    <a href="#"> {{object.created_user}}</a></p>
-	   {{ object.body|markdown}}
-	  </div>
-	   {% block link %}
-	   {% for contact in contacts %}
-    {# Each "contact" is a Contact model object. #}
-    {{ contact.full_name|upper }}<br />
-    ...
-{% endfor %}
-{% endblock %}  
+        <h2><a class="blog-post-title"><a href="{% url "entry_view" object.id %}">{{object.title}}</a></h2>
+	    {% for tag in object.tags.all %}
+	       <a href="{% url 'tag_entry_view' tag.slug %}">{{ tag.name }}</a>
+	    {% endfor %}
+	    <p class="blog-post-meta"> 
+	       {{object.posted}} <a href="#"> {{object.created_user}}</a>
```

Comment:
Look at how to format the date ``object.posted`` via django date template tags:

- https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#date

You can also use:

- https://docs.djangoproject.com/en/1.8/ref/contrib/humanize/#naturaltime